### DISCUSSION TITLE
Is it possible to use Class B (/16) instead of a Class C (/24) subnets?

### DISCUSSION THREAD

**Discussion by pleibling** [wg-easy/wg-easy#1477] [Q&A] [answered]
Hello,

one of my customer want to use more than 250 addresses, is it possible to use /16 then /24?

e.g. - WG_DEFAULT_ADDRESS=10.8.**x.x** instead of - WG_DEFAULT_ADDRESS=10.8.0.**x**?

**Comment by kaaax0815** [comment, marked as answer]
Not currently. Will be implemented with #1333

**Reply by pleibling** [reply]
Is there an public Beta with this feature implemented?

Thanks a lot @kaaax0815

**Reply by kaaax0815** [reply]
There is the `development` Tag. But its not even an alpha. Its very experimental right now, so I suggest to not use it
